webapp/tests/karma/unit/store/index.js

```diff
@@ -98,226 +137,292 @@ describe('Store', function() {
       chai.expect(state).to.not.equal(initialState);
-      chai.expect(globalState.selected).to.not.equal(selectors.getGlobalState(initialState).selected);
-      chai.expect(globalState).to.deep.equal({ selected: [{ formatted }] });
+      chai.expect(analyticsState).to.deep.equal({ selected });
     });
+  });
 
-    it('sets loadingSelectedChildren', () => {
-      const initialState = createGlobalState({ loadingSelectedChildren: false });
+  describe('Contacts', () => {
+    it('sets selected contact', () => {
+      const initialState = createContactsState({ selected: null });
       setupStore(initialState);
-      globalActions.setLoadingSelectedChildren(true);
+      const selected = {};
+      contactsActions.setSelectedContact(selected);
       const state = getState();
-      const globalState = selectors.getGlobalState(state);
+      const contactsState = selectors.getContactsState(state);
       chai.expect(state).to.not.equal(initialState);
-      chai.expect(globalState).to.deep.equal({ loadingSelectedChildren: true });
+      chai.expect(contactsState).to.deep.equal({ selected });
     });
 
-    it('sets loadingSelectedReports', () => {
-      const initialState = createGlobalState({ loadingSelectedReports: false });
+    it('updates selected contact', () => {
+      const initialState = createContactsState({ selected: { doc: '1' } });
       setupStore(initialState);
-      globalActions.setLoadingSelectedReports(true);
+      const selected = { doc: '2' };
+      contactsActions.updateSelectedContact(selected);
       const state = getState();
-      const globalState = selectors.getGlobalState(state);
+      const contactsState = selectors.getContactsState(state);
       chai.expect(state).to.not.equal(initialState);
-      chai.expect(globalState).to.deep.equal({ loadingSelectedReports: true });
+      chai.expect(contactsState.selected).to.not.equal(selectors.getContactsState(initialState).selected);
+      chai.expect(contactsState).to.deep.equal({ selected });
     });
 
-    it('loads selected children', (done) => {
+    it('sets loadingSelectedContactChildren', () => {
+      const initialState = createContactsState({ loadingSelectedChildren: false });
+      setupStore(initialState);
+      contactsActions.setLoadingSelectedContactChildren(true);
+      const state = getState();
+      const contactsState = selectors.getContactsState(state);
+      chai.expect(state).to.not.equal(initialState);
+      chai.expect(contactsState).to.deep.equal({ loadingSelectedChildren: true });
+    });
+
+    it('sets loadingSelectedContactReports', () => {
+      const initialState = createContactsState({ loadingSelectedReports: false });
+      setupStore(initialState);
+      contactsActions.setLoadingSelectedContactReports(true);
+      const state = getState();
+      const contactsState = selectors.getContactsState(state);
+      chai.expect(state).to.not.equal(initialState);
```

Comment:
`const { expect } = require('chai');` ?

Reply:
Yeah that makes a lot more sense to me but not what I've seen across a lot of the codebase, happy to have that as the new standard moving forward though.